pacttesting/testing.go

```diff
@@ -42,10 +42,9 @@ var (
 	pactServers = make(map[string]*MockServer)
 )
 
-var defaultRetryOptions = []retry.RetryOption{
-	retry.MaxTries(150000),
-	retry.Sleep(200 * time.Millisecond),
-	retry.Timeout(3 * time.Minute),
+var defaultRetryOptions = []retry.Option{
+	retry.Attempts(15),
+	retry.Delay(200 * time.Millisecond),
```

Comment:
avast/retry-go does not include a timeout, however using the power of maths™️ I adjusted the attempts and delay to reflect what the timeout would have been